src/iconv.jl

```diff
@@ -70,7 +79,7 @@ function iconv!(cd::Ptr{Void}, inbuf::Vector{UInt8}, outbuf::Vector{UInt8},
                 (Ptr{Void}, Ptr{Ptr{UInt8}}, Ref{Csize_t}, Ptr{Ptr{UInt8}}, Ref{Csize_t}),
                 cd, inbufptr, inbytesleft, outbufptr, outbytesleft)
 
-    if ret == reinterpret(Csize_t, -1)
+    if ret == (-1)%Csize_t
```

Comment:
I'd prefer a space here. Also, please remove the parentheses. To avoid any possible ambiguity, you would need two pairs of them anyway.